tests/integration/components/angle-bracket-invocation-test.js

```diff
@@ -121,6 +121,31 @@ module('Integration | Component | angle-bracket-invocation', function(hooks) {
       assert.dom('[data-foo="bar"]').exists();
     });
 
+    test('with attributes - has attributeBindings', async function(assert) {
```

Comment:
FWIW, I wouldn't expect `attributeBindings` to have any effect here (since the invocation doesn't pass `@data-foo`).

Reply:
Yes, indeed. But it should still have that attribute applied (not through `attributeBindings` but through the angle bracket invocation with that attribute), but the presence of that `attributeBindings` seems to make this fail! So it *does* have an (unwanted) effect here!